content/reference/admin/private_locations/private_packages/index.md

```diff
@@ -9,71 +9,140 @@ weight: 22033
 
 # Introduction
 
-Using Private Locations allows you to run your simulations in your private cloud. Now, what about your simulations' packages? They may hold private information, so you may want to also store them in your private cloud.
+Private Locations facilitate running simulations in your dedicated cloud environment. 
+Ensure secure storage of sensitive simulation packages in your private cloud. 
 
-Your Control Plane has a private repository feature, allowing you to manage your simulation packages privately, you just have to enable it.
+The Control Plane offers a private repository; enable it for confidential package management!
```

Comment:
There is some inconsistent capitalization of the terms "control plane", "private locations", and "private packages". The control plane should always be lowercase as it is a widely used term in network traffic routing.  A private package or location is lowercase, but if we are referring to the Gatling features, is should be Private Packages and Private Locations (both plural and capitalized). WDYT?